src/test/incremental/hashes/indexing_expressions.rs

```diff
@@ -153,5 +153,5 @@ fn exclusive_to_inclusive_range(slice: &[u32]) -> &[u32] {
 #[rustc_metadata_clean(cfg="cfail2")]
 #[rustc_metadata_clean(cfg="cfail3")]
 fn exclusive_to_inclusive_range(slice: &[u32]) -> &[u32] {
-    &slice[3...7]
+    &slice[3..=7]
```

Comment:
Sigh, the syntax is incredibly ugly.

Reply:
Now I see all these examples, I am more convinced than ever that this syntax is a bad idea.